splink/connected_components.py

```diff
@@ -281,15 +344,38 @@ def solve_connected_components(
     and generates the final SQL for our algorithm.
 
     The code will continue to loop until a solution is identified.
+
+    Returns:
```

Comment:
We want these in google docstring style.  Easiest way to do this is use [autodocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) in VS code, making sure you set the default style to google